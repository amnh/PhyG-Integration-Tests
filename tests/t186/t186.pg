--build(character, block), inputting prefasta file
set(seed:1634561640)
read(prefasta: "chel-prealigned.fas")
read(prefasta: "metazoa-aa-1.fas")
build(character, block, replicates:3)
report("t186.csv", data, overwrite)
report("t186.tre", newick, graphs, overwrite)