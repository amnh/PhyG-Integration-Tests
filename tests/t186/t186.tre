(Alentus:56.0,(Aportus:64.5,(Aplysia_californica:91.5,(((Venerupis_decussatus:132.0,(Limulus:70.5,(Amblypygid:66.0,Littorina_littorea:77.5)HTU58:8.0)HTU55:8.0)HTU56:5.5,(Mastigoproctus:63.5,Yoldia_limatula:78.0)HTU59:13.0)HTU57:5.5,((Gea:92.5,(Vonones:68.5,(Thermobius:81.5,(Artemia:101.5,(Venerupis_philippinarum:101.5,(Ilyanassa_obsoleta:78.0,(Crassostrea_virginica:115.0,(Perotrochus_lucaya:77.0,(Pauroctonus:70.0,Scaphopods:79.5)HTU44:6.5)HTU39:4.0)HTU34:4.5)HTU36:9.0)HTU40:13.0)HTU45:8.0)HTU47:11.0)HTU50:11.0)HTU52:4.5,(Hypochilus:67.0,(Rhiphicephalus:95.5,(Chanbria:74.0,(Americhernus:117.5,(Lingula_anatina:100.5,((Centruoides:70.0,(Lottia_gigantea:84.0,(Siphonaria_pectinata:102.0,(Mytilus_californianus:84.5,Thelichoris:65.5)HTU43:17.5)HTU38:8.0)HTU33:8.5)HTU35:7.5,(Lymnaea_stagnalis:82.0,(Hadrurus:69.5,Helobdella_robusta:82.5)HTU49:9.0)HTU41:16.0)HTU37:11.5)HTU42:6.0)HTU46:9.5)HTU48:5.0)HTU51:8.0)HTU53:3.5)HTU62:3.5)HTU54:29.5)HTU60:26.0)HTU61:3.0)HTU32[787.0];
(Alentus:56.0,(Aportus:64.5,(Aplysia_californica:91.0,(((Venerupis_decussatus:133.0,((Crassostrea_virginica:90.0,Limulus:63.5)HTU53:31.5,(Amblypygid:71.0,(Centruoides:65.5,Littorina_littorea:78.5)HTU59:10.0)HTU57:8.0)HTU54:8.0)HTU55:4.5,(Mastigoproctus:63.0,Yoldia_limatula:77.5)HTU58:13.0)HTU56:6.0,((Gea:92.0,(Vonones:68.5,(Thermobius:81.5,(Artemia:101.5,(Venerupis_philippinarum:101.5,(Ilyanassa_obsoleta:76.5,(Perotrochus_lucaya:75.5,(Pauroctonus:70.0,Scaphopods:79.5)HTU42:8.0)HTU37:3.0)HTU34:10.0)HTU38:13.0)HTU43:8.0)HTU45:11.0)HTU48:11.0)HTU50:4.5,(Hypochilus:67.0,(Rhiphicephalus:97.5,(Chanbria:73.0,(Americhernus:119.0,(Lingula_anatina:100.0,((Lottia_gigantea:83.0,(Siphonaria_pectinata:101.0,(Mytilus_californianus:83.5,Thelichoris:63.5)HTU41:17.5)HTU36:8.0)HTU33:8.5,(Lymnaea_stagnalis:81.5,(Hadrurus:71.0,Helobdella_robusta:82.0)HTU47:9.0)HTU39:18.0)HTU35:10.0)HTU40:5.0)HTU44:8.0)HTU46:5.0)HTU49:6.0)HTU51:3.0)HTU62:4.0)HTU52:30.0)HTU60:26.0)HTU61:3.0)HTU32[786.0];
(Alentus:56.0,(Aportus:64.5,(Aplysia_californica:91.0,(((Venerupis_decussatus:132.5,((Crassostrea_virginica:89.5,Limulus:65.5)HTU54:31.5,(Amblypygid:66.0,Littorina_littorea:78.0)HTU58:9.0)HTU55:8.0)HTU56:5.0,(Mastigoproctus:63.5,Yoldia_limatula:77.5)HTU59:12.5)HTU57:6.0,((Gea:92.5,(Vonones:68.5,(Thermobius:81.5,(Artemia:101.5,(Venerupis_philippinarum:101.5,(Ilyanassa_obsoleta:76.5,(Perotrochus_lucaya:75.5,(Pauroctonus:70.0,Scaphopods:79.5)HTU43:8.0)HTU38:3.0)HTU35:10.0)HTU39:13.0)HTU44:8.0)HTU46:11.0)HTU49:11.0)HTU51:4.5,(Hypochilus:67.0,(Rhiphicephalus:95.5,(Chanbria:74.0,(Americhernus:117.5,(Lingula_anatina:100.5,((Centruoides:70.0,(Lottia_gigantea:84.0,(Siphonaria_pectinata:102.0,(Mytilus_californianus:84.5,Thelichoris:65.5)HTU42:17.5)HTU37:8.0)HTU33:8.5)HTU34:7.5,(Lymnaea_stagnalis:82.0,(Hadrurus:69.5,Helobdella_robusta:82.5)HTU48:9.0)HTU40:16.0)HTU36:11.5)HTU41:6.0)HTU45:9.5)HTU47:5.0)HTU50:8.0)HTU52:3.5)HTU62:4.0)HTU53:29.5)HTU60:26.0)HTU61:3.0)HTU32[785.0];
(Alentus:58.5,(Aportus:66.0,(Aplysia_californica:89.0,(((Venerupis_decussatus:119.5,(Lingula_anatina:87.5,(Limulus:70.0,(Centruoides:66.5,Littorina_littorea:82.0)HTU59:10.0)HTU54:16.5)HTU55:2.5)HTU56:17.0,(Mastigoproctus:62.5,Yoldia_limatula:78.0)HTU58:11.0)HTU57:13.5,((Hypochilus:68.0,(Rhiphicephalus:97.5,(Chanbria:73.0,(Americhernus:119.0,((Lottia_gigantea:82.5,(Siphonaria_pectinata:100.0,(Mytilus_californianus:83.0,Thelichoris:63.5)HTU41:18.0)HTU37:8.0)HTU33:8.5,(Lymnaea_stagnalis:80.5,(Hadrurus:71.0,Helobdella_robusta:82.0)HTU47:10.0)HTU40:17.0)HTU36:7.0)HTU44:8.0)HTU46:5.0)HTU49:6.0)HTU51:4.0,(Amblypygid:72.5,(Gea:94.0,(Vonones:68.0,(Thermobius:81.0,(Artemia:101.0,(Venerupis_philippinarum:106.5,(Ilyanassa_obsoleta:78.0,(Crassostrea_virginica:115.5,(Perotrochus_lucaya:77.0,(Pauroctonus:70.0,Scaphopods:79.5)HTU42:6.5)HTU38:4.0)HTU34:4.0)HTU35:4.5)HTU39:13.0)HTU43:8.0)HTU45:11.0)HTU48:9.5)HTU50:3.5)HTU52:5.5)HTU62:5.0)HTU53:25.5)HTU60:25.0)HTU61:3.0)HTU32[784.0];
(Alentus:58.5,(Aportus:66.0,(Aplysia_californica:89.0,(((Venerupis_decussatus:119.5,(Lingula_anatina:87.5,(Limulus:70.0,(Centruoides:66.5,Littorina_littorea:82.0)HTU59:10.0)HTU54:16.5)HTU55:2.5)HTU56:17.0,(Mastigoproctus:62.5,Yoldia_limatula:78.0)HTU58:11.0)HTU57:13.5,((Hypochilus:68.0,(Rhiphicephalus:97.5,(Chanbria:73.0,(Americhernus:119.0,((Lottia_gigantea:82.5,(Siphonaria_pectinata:100.0,(Mytilus_californianus:83.0,Thelichoris:63.5)HTU41:18.0)HTU37:8.0)HTU33:8.5,(Lymnaea_stagnalis:80.5,(Hadrurus:71.0,Helobdella_robusta:82.0)HTU47:10.0)HTU40:17.0)HTU36:7.0)HTU44:8.0)HTU46:5.0)HTU49:6.0)HTU51:4.0,(Amblypygid:72.5,(Gea:94.0,(Vonones:68.0,(Thermobius:81.0,(Artemia:101.0,(Venerupis_philippinarum:106.5,(Ilyanassa_obsoleta:78.0,(Crassostrea_virginica:115.5,(Perotrochus_lucaya:77.0,(Pauroctonus:70.0,Scaphopods:79.5)HTU42:6.5)HTU38:4.0)HTU34:4.0)HTU35:4.5)HTU39:13.0)HTU43:8.0)HTU45:11.0)HTU48:9.5)HTU50:3.5)HTU52:5.5)HTU62:5.0)HTU53:25.5)HTU60:25.0)HTU61:3.0)HTU32[784.0];
(Alentus:58.5,(Aportus:66.0,(Aplysia_californica:89.0,(((Venerupis_decussatus:120.0,(Lingula_anatina:88.0,((Crassostrea_virginica:90.0,Limulus:63.0)HTU53:32.0,(Centruoides:65.5,Littorina_littorea:78.5)HTU59:10.5)HTU54:20.5)HTU55:2.5)HTU56:16.0,(Mastigoproctus:62.5,Yoldia_limatula:78.0)HTU58:11.0)HTU57:13.5,((Hypochilus:68.0,(Rhiphicephalus:97.5,(Chanbria:73.0,(Americhernus:119.0,((Lottia_gigantea:82.5,(Siphonaria_pectinata:100.0,(Mytilus_californianus:83.0,Thelichoris:63.5)HTU40:18.0)HTU36:8.0)HTU33:8.5,(Lymnaea_stagnalis:80.5,(Hadrurus:71.0,Helobdella_robusta:82.0)HTU46:10.0)HTU39:17.0)HTU35:7.0)HTU43:8.0)HTU45:5.0)HTU48:6.0)HTU50:4.5,(Amblypygid:73.0,(Gea:94.5,(Vonones:68.5,(Thermobius:81.5,(Artemia:101.5,(Venerupis_philippinarum:106.5,(Ilyanassa_obsoleta:76.5,(Perotrochus_lucaya:75.5,(Pauroctonus:70.0,Scaphopods:79.5)HTU41:8.0)HTU37:3.0)HTU34:5.0)HTU38:13.0)HTU42:8.0)HTU44:11.0)HTU47:9.5)HTU49:3.5)HTU51:5.5)HTU62:5.0)HTU52:25.5)HTU60:25.0)HTU61:3.0)HTU32[783.0];
(Alentus:58.0,(Aportus:66.5,(Aplysia_californica:89.0,(((Venerupis_decussatus:120.5,(Lingula_anatina:88.5,((Crassostrea_virginica:90.5,Limulus:62.5)HTU52:31.0,(Amblypygid:70.0,(Centruoides:64.5,Littorina_littorea:79.0)HTU59:10.0)HTU57:7.5)HTU53:20.0)HTU54:2.5)HTU55:16.0,(Mastigoproctus:62.5,Yoldia_limatula:78.0)HTU58:13.0)HTU56:12.0,((Gea:92.0,(Vonones:68.5,(Thermobius:81.5,(Artemia:101.5,(Venerupis_philippinarum:106.5,(Ilyanassa_obsoleta:76.5,(Perotrochus_lucaya:75.5,(Pauroctonus:70.0,Scaphopods:79.5)HTU41:8.0)HTU37:3.0)HTU34:5.0)HTU38:13.0)HTU42:8.0)HTU44:11.0)HTU47:11.0)HTU49:5.5,(Hypochilus:67.0,(Rhiphicephalus:97.5,(Chanbria:73.0,(Americhernus:119.0,((Lottia_gigantea:82.5,(Siphonaria_pectinata:100.0,(Mytilus_californianus:83.0,Thelichoris:63.5)HTU40:18.0)HTU36:8.0)HTU33:8.5,(Lymnaea_stagnalis:80.5,(Hadrurus:71.0,Helobdella_robusta:82.0)HTU46:10.0)HTU39:17.0)HTU35:7.0)HTU43:8.0)HTU45:5.0)HTU48:6.0)HTU50:4.0)HTU62:3.5)HTU51:26.5)HTU60:26.0)HTU61:3.0)HTU32[782.0];
(Alentus:58.0,(Aportus:66.5,(Aplysia_californica:89.0,(((Venerupis_decussatus:119.5,(Lingula_anatina:87.5,(Limulus:70.5,(Amblypygid:66.0,Littorina_littorea:82.0)HTU58:8.0)HTU54:16.0)HTU55:2.5)HTU56:17.5,(Mastigoproctus:63.0,Yoldia_limatula:78.0)HTU59:12.5)HTU57:12.0,((Gea:92.0,(Vonones:68.0,(Thermobius:81.0,(Artemia:101.0,(Venerupis_philippinarum:106.5,(Ilyanassa_obsoleta:78.0,(Crassostrea_virginica:115.5,(Perotrochus_lucaya:77.0,(Pauroctonus:70.0,Scaphopods:79.5)HTU43:6.5)HTU39:4.0)HTU34:4.0)HTU36:4.5)HTU40:13.0)HTU44:8.0)HTU46:11.0)HTU49:11.0)HTU51:5.5,(Hypochilus:67.0,(Rhiphicephalus:95.5,(Chanbria:74.0,(Americhernus:117.5,((Centruoides:70.0,(Lottia_gigantea:83.5,(Siphonaria_pectinata:101.0,(Mytilus_californianus:84.0,Thelichoris:65.5)HTU42:18.0)HTU38:8.0)HTU33:8.5)HTU35:7.5,(Lymnaea_stagnalis:81.0,(Hadrurus:69.5,Helobdella_robusta:82.5)HTU48:10.0)HTU41:15.0)HTU37:9.5)HTU45:9.5)HTU47:5.0)HTU50:8.0)HTU52:4.0)HTU62:3.5)HTU53:26.0)HTU60:26.0)HTU61:3.0)HTU32[782.0];
(Alentus:58.0,(Aportus:66.5,(Aplysia_californica:89.0,(((Venerupis_decussatus:120.0,(Lingula_anatina:88.0,(Limulus:63.5,Littorina_littorea:82.5)HTU55:15.5)HTU56:2.5)HTU57:20.5,(Mastigoproctus:63.5,Yoldia_limatula:78.0)HTU59:13.5)HTU58:12.5,((Hypochilus:68.0,(Rhiphicephalus:95.0,(Chanbria:74.5,(Americhernus:117.5,((Centruoides:70.0,(Lottia_gigantea:83.5,(Siphonaria_pectinata:101.0,(Mytilus_californianus:84.0,Thelichoris:65.5)HTU42:18.0)HTU38:8.0)HTU33:8.5)HTU35:7.5,(Lymnaea_stagnalis:81.0,(Hadrurus:69.5,Helobdella_robusta:82.5)HTU48:10.0)HTU41:15.0)HTU37:9.5)HTU45:9.0)HTU47:5.5)HTU50:7.5)HTU52:4.5,(Amblypygid:72.0,(Gea:94.5,(Vonones:68.5,(Thermobius:81.0,(Artemia:101.0,(Venerupis_philippinarum:106.5,(Ilyanassa_obsoleta:78.0,(Crassostrea_virginica:115.5,(Perotrochus_lucaya:77.0,(Pauroctonus:70.0,Scaphopods:79.5)HTU43:6.5)HTU39:4.0)HTU34:4.0)HTU36:4.5)HTU40:13.0)HTU44:8.0)HTU46:10.5)HTU49:9.5)HTU51:3.5)HTU53:5.5)HTU62:4.0)HTU54:24.5)HTU60:26.0)HTU61:3.0)HTU32[782.0];
(Alentus:58.0,(Aportus:66.5,(Aplysia_californica:89.0,(((Venerupis_decussatus:120.0,(Lingula_anatina:88.0,(Limulus:63.5,Littorina_littorea:82.5)HTU55:15.5)HTU56:2.5)HTU57:20.5,(Mastigoproctus:63.5,Yoldia_limatula:78.0)HTU59:13.5)HTU58:12.5,((Hypochilus:68.0,(Rhiphicephalus:95.0,(Chanbria:74.5,(Americhernus:117.5,((Centruoides:70.0,(Lottia_gigantea:83.5,(Siphonaria_pectinata:101.0,(Mytilus_californianus:84.0,Thelichoris:65.5)HTU42:18.0)HTU38:8.0)HTU33:8.5)HTU35:7.5,(Lymnaea_stagnalis:81.0,(Hadrurus:69.5,Helobdella_robusta:82.5)HTU48:10.0)HTU41:15.0)HTU37:9.5)HTU45:9.0)HTU47:5.5)HTU50:7.5)HTU52:4.5,(Amblypygid:72.0,(Gea:94.5,(Vonones:68.5,(Thermobius:81.0,(Artemia:101.0,(Venerupis_philippinarum:106.5,(Ilyanassa_obsoleta:78.0,(Crassostrea_virginica:115.5,(Perotrochus_lucaya:77.0,(Pauroctonus:70.0,Scaphopods:79.5)HTU43:6.5)HTU39:4.0)HTU34:4.0)HTU36:4.5)HTU40:13.0)HTU44:8.0)HTU46:10.5)HTU49:9.5)HTU51:3.5)HTU53:5.5)HTU62:4.0)HTU54:24.5)HTU60:26.0)HTU61:3.0)HTU32[782.0];
