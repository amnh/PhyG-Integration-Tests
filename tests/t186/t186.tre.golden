(Alentus:69.0,((Americhernus:111.0,(Venerupis_decussatus:121.0,(Chanbria:75.0,(Aplysia_californica:112.5,(Venerupis_philippinarum:97.0,(Hadrurus:84.5,(Artemia:110.5,(Gea:86.5,(Yoldia_limatula:80.0,(Helobdella_robusta:90.0,(Crassostrea_virginica:113.5,(Hypochilus:64.5,(Rhiphicephalus:100.0,(Limulus:69.5,(Siphonaria_pectinata:107.0,(Perotrochus_lucaya:76.5,(Scaphopods:86.0,(Thelichoris:71.0,(Aportus:96.0,(Centruoides:69.5,(Lingula_anatina:106.5,(Lymnaea_stagnalis:84.5,(Thermobius:90.5,(Littorina_littorea:75.0,(Mastigoproctus:72.0,(Ilyanassa_obsoleta:74.5,Pauroctonus:67.5)HTU33:8.0)HTU34:4.0)HTU35:6.0)HTU36:3.5)HTU37:2.5)HTU38:6.0)HTU39:7.0)HTU40:7.5)HTU41:4.0)HTU42:2.5)HTU43:5.0)HTU44:4.5)HTU45:3.0)HTU46:5.5)HTU47:6.0)HTU48:4.0)HTU49:9.5)HTU50:7.5)HTU51:9.5)HTU52:4.5)HTU53:5.0)HTU54:2.5)HTU55:7.5)HTU56:11.5)HTU57:7.0)HTU58:6.5,(Mytilus_californianus:90.0,(Lottia_gigantea:83.0,(Amblypygid:72.5,Vonones:70.5)HTU61:5.5)HTU60:14.5)HTU62:4.0)HTU59:13.5)HTU32[815.0];
(Alentus:69.0,((Americhernus:111.0,(Venerupis_decussatus:121.0,(Chanbria:75.0,(Aplysia_californica:112.5,(Venerupis_philippinarum:97.0,(Hadrurus:84.5,(Artemia:110.5,(Gea:86.5,(Yoldia_limatula:80.0,(Helobdella_robusta:90.0,(Crassostrea_virginica:113.5,(Hypochilus:64.5,(Rhiphicephalus:100.0,(Limulus:69.5,(Siphonaria_pectinata:107.0,(Perotrochus_lucaya:76.5,(Scaphopods:86.0,(Thelichoris:71.0,(Aportus:96.0,(Centruoides:69.5,(Lingula_anatina:106.5,(Lymnaea_stagnalis:84.5,(Thermobius:90.5,(Littorina_littorea:75.0,(Mastigoproctus:72.0,(Ilyanassa_obsoleta:74.5,Pauroctonus:67.5)HTU33:8.0)HTU34:4.0)HTU35:6.0)HTU36:3.5)HTU37:2.5)HTU38:6.0)HTU39:7.0)HTU40:7.5)HTU41:4.0)HTU42:2.5)HTU43:5.0)HTU44:4.5)HTU45:3.0)HTU46:5.5)HTU47:6.0)HTU48:4.0)HTU49:9.5)HTU50:7.5)HTU51:9.5)HTU52:4.5)HTU53:5.0)HTU54:2.5)HTU55:7.5)HTU56:11.5)HTU57:7.0)HTU58:6.5,(Mytilus_californianus:90.0,(Lottia_gigantea:83.0,(Amblypygid:72.5,Vonones:70.5)HTU61:5.5)HTU60:14.5)HTU62:4.0)HTU59:13.5)HTU32[815.0];
(Alentus:69.0,((Venerupis_decussatus:122.5,(Chanbria:75.5,(Aplysia_californica:113.0,(Venerupis_philippinarum:97.5,(Hadrurus:85.0,(Artemia:111.0,(Gea:86.5,(Yoldia_limatula:80.0,(Helobdella_robusta:90.0,(Crassostrea_virginica:113.5,(Hypochilus:64.5,(Rhiphicephalus:100.0,(Limulus:69.5,(Siphonaria_pectinata:107.0,(Perotrochus_lucaya:76.5,(Scaphopods:86.0,(Thelichoris:71.0,(Aportus:96.0,(Centruoides:69.5,(Lingula_anatina:106.5,(Lymnaea_stagnalis:84.5,(Thermobius:90.5,(Littorina_littorea:75.0,(Mastigoproctus:72.0,(Ilyanassa_obsoleta:74.5,Pauroctonus:67.5)HTU33:8.0)HTU34:4.0)HTU35:6.0)HTU36:3.5)HTU37:2.5)HTU38:6.0)HTU39:7.0)HTU40:7.5)HTU41:4.0)HTU42:2.5)HTU43:5.0)HTU44:4.5)HTU45:3.0)HTU46:5.5)HTU47:6.0)HTU48:4.0)HTU49:9.5)HTU50:7.5)HTU51:8.5)HTU52:5.0)HTU53:5.0)HTU54:2.5)HTU55:6.5)HTU56:11.5)HTU57:8.5,(Mytilus_californianus:90.0,(Americhernus:115.5,(Lottia_gigantea:83.0,(Amblypygid:71.5,Vonones:71.5)HTU61:5.5)HTU59:3.5)HTU60:15.0)HTU62:5.5)HTU58:13.0)HTU32[815.0];
(Alentus:69.0,((Americhernus:111.0,(Venerupis_decussatus:121.0,(Chanbria:75.0,(Aplysia_californica:112.5,(Venerupis_philippinarum:97.0,(Hadrurus:84.5,(Artemia:110.5,(Gea:86.5,(Yoldia_limatula:80.0,(Helobdella_robusta:90.0,(Crassostrea_virginica:113.5,(Hypochilus:64.5,(Rhiphicephalus:100.0,(Limulus:69.5,(Siphonaria_pectinata:107.0,(Perotrochus_lucaya:76.5,(Scaphopods:86.0,(Thelichoris:71.0,(Aportus:96.0,(Centruoides:69.5,(Lingula_anatina:106.5,(Lymnaea_stagnalis:84.5,(Thermobius:90.5,(Littorina_littorea:75.0,(Mastigoproctus:72.0,(Ilyanassa_obsoleta:74.5,Pauroctonus:67.5)HTU33:8.0)HTU34:4.0)HTU35:6.0)HTU36:3.5)HTU37:2.5)HTU38:6.0)HTU39:7.0)HTU40:7.5)HTU41:4.0)HTU42:2.5)HTU43:5.0)HTU44:4.5)HTU45:3.0)HTU46:5.5)HTU47:6.0)HTU48:4.0)HTU49:9.5)HTU50:7.5)HTU51:9.5)HTU52:4.5)HTU53:5.0)HTU54:2.5)HTU55:7.5)HTU56:11.5)HTU57:7.0)HTU58:6.5,(Mytilus_californianus:90.0,(Lottia_gigantea:83.0,(Amblypygid:72.5,Vonones:70.5)HTU61:5.5)HTU60:14.5)HTU62:4.0)HTU59:13.5)HTU32[815.0];
(Alentus:69.0,((Venerupis_decussatus:122.5,(Chanbria:76.0,(Aplysia_californica:113.0,(Venerupis_philippinarum:97.5,(Hadrurus:85.0,(Artemia:111.0,(Gea:86.5,(Yoldia_limatula:80.0,(Helobdella_robusta:90.0,(Crassostrea_virginica:113.5,(Hypochilus:64.5,(Rhiphicephalus:99.5,(Limulus:69.0,(Siphonaria_pectinata:107.0,(Perotrochus_lucaya:76.5,(Scaphopods:86.0,(Thelichoris:71.5,(Aportus:95.0,(Centruoides:73.0,(Lingula_anatina:106.5,(Lymnaea_stagnalis:84.5,(Thermobius:89.5,(Littorina_littorea:76.5,(Mastigoproctus:70.5,(Ilyanassa_obsoleta:76.0,(Amblypygid:70.5,Pauroctonus:77.5)HTU34:3.5)HTU33:3.5)HTU35:4.5)HTU36:5.0)HTU37:4.0)HTU38:2.5)HTU39:5.5)HTU40:4.5)HTU41:8.0)HTU42:4.0)HTU43:2.5)HTU44:5.0)HTU45:4.5)HTU46:3.0)HTU47:6.0)HTU48:6.0)HTU49:4.0)HTU50:9.5)HTU51:7.5)HTU52:8.5)HTU53:5.0)HTU54:5.0)HTU55:2.5)HTU56:6.5)HTU57:10.5)HTU58:8.5,(Mytilus_californianus:90.0,(Americhernus:116.5,(Lottia_gigantea:83.0,Vonones:65.0)HTU60:7.0)HTU61:15.0)HTU62:5.5)HTU59:14.0)HTU32[813.0];
(Alentus:69.0,((Venerupis_decussatus:122.5,(Chanbria:76.0,(Aplysia_californica:113.0,(Venerupis_philippinarum:97.5,(Hadrurus:85.0,(Artemia:111.0,(Gea:86.5,(Yoldia_limatula:80.0,(Helobdella_robusta:90.0,(Crassostrea_virginica:113.5,(Hypochilus:64.5,(Rhiphicephalus:99.5,(Limulus:69.0,(Siphonaria_pectinata:107.0,(Perotrochus_lucaya:76.5,(Scaphopods:86.0,(Thelichoris:71.5,(Aportus:95.0,(Centruoides:73.0,(Lingula_anatina:106.5,(Lymnaea_stagnalis:84.5,(Thermobius:89.5,(Littorina_littorea:76.5,(Mastigoproctus:70.5,(Ilyanassa_obsoleta:76.0,(Amblypygid:70.5,Pauroctonus:77.5)HTU34:3.5)HTU33:3.5)HTU35:4.5)HTU36:5.0)HTU37:4.0)HTU38:2.5)HTU39:5.5)HTU40:4.5)HTU41:8.0)HTU42:4.0)HTU43:2.5)HTU44:5.0)HTU45:4.5)HTU46:3.0)HTU47:6.0)HTU48:6.0)HTU49:4.0)HTU50:9.5)HTU51:7.5)HTU52:8.5)HTU53:5.0)HTU54:5.0)HTU55:2.5)HTU56:6.5)HTU57:10.5)HTU58:8.5,(Mytilus_californianus:90.0,(Americhernus:116.5,(Lottia_gigantea:83.0,Vonones:65.0)HTU60:7.0)HTU61:15.0)HTU62:5.5)HTU59:14.0)HTU32[813.0];
(Alentus:71.0,((Venerupis_decussatus:115.5,(Chanbria:74.5,(Aplysia_californica:113.5,(Venerupis_philippinarum:97.0,(Hadrurus:85.0,(Artemia:111.0,(Gea:86.5,(Yoldia_limatula:80.0,(Helobdella_robusta:90.0,(Crassostrea_virginica:114.0,(Hypochilus:64.5,(Rhiphicephalus:100.0,(Limulus:69.5,(Lottia_gigantea:80.5,(Siphonaria_pectinata:109.0,(Perotrochus_lucaya:76.5,(Scaphopods:86.0,(Thelichoris:71.0,(Aportus:96.0,(Centruoides:69.5,(Lingula_anatina:106.5,(Lymnaea_stagnalis:84.5,(Thermobius:90.5,(Littorina_littorea:75.0,(Mastigoproctus:72.0,(Ilyanassa_obsoleta:74.5,Pauroctonus:67.5)HTU33:8.0)HTU34:4.0)HTU35:6.0)HTU36:3.5)HTU37:2.5)HTU38:6.0)HTU39:7.0)HTU40:7.5)HTU41:4.0)HTU42:2.5)HTU43:3.5)HTU44:3.0)HTU45:5.0)HTU46:3.0)HTU47:5.5)HTU48:6.5)HTU49:3.5)HTU50:9.5)HTU51:8.0)HTU52:8.5)HTU53:5.0)HTU54:5.0)HTU55:2.5)HTU56:7.5)HTU57:18.0)HTU58:11.5,(Mytilus_californianus:78.5,(Americhernus:114.5,(Amblypygid:70.5,Vonones:70.5)HTU61:6.5)HTU60:3.0)HTU62:5.0)HTU59:13.0)HTU32[794.0];
(Alentus:71.0,((Venerupis_decussatus:115.5,(Chanbria:74.5,(Aplysia_californica:113.5,(Venerupis_philippinarum:97.0,(Hadrurus:85.0,(Artemia:111.0,(Gea:86.5,(Yoldia_limatula:80.0,(Helobdella_robusta:90.0,(Crassostrea_virginica:114.0,(Hypochilus:64.5,(Rhiphicephalus:100.0,(Limulus:69.5,(Lottia_gigantea:80.5,(Siphonaria_pectinata:109.0,(Perotrochus_lucaya:76.5,(Scaphopods:86.0,(Thelichoris:71.0,(Aportus:96.0,(Centruoides:69.5,(Lingula_anatina:106.5,(Lymnaea_stagnalis:84.5,(Thermobius:90.5,(Littorina_littorea:75.0,(Mastigoproctus:72.0,(Ilyanassa_obsoleta:74.5,Pauroctonus:67.5)HTU33:8.0)HTU34:4.0)HTU35:6.0)HTU36:3.5)HTU37:2.5)HTU38:6.0)HTU39:7.0)HTU40:7.5)HTU41:4.0)HTU42:2.5)HTU43:3.5)HTU44:3.0)HTU45:5.0)HTU46:3.0)HTU47:5.5)HTU48:6.5)HTU49:3.5)HTU50:9.5)HTU51:8.0)HTU52:8.5)HTU53:5.0)HTU54:5.0)HTU55:2.5)HTU56:7.5)HTU57:18.0)HTU58:11.5,(Mytilus_californianus:78.5,(Americhernus:114.5,(Amblypygid:70.5,Vonones:70.5)HTU61:6.5)HTU60:3.0)HTU62:5.0)HTU59:13.0)HTU32[794.0];
(Alentus:71.0,((Venerupis_decussatus:115.5,(Chanbria:75.0,(Aplysia_californica:113.5,(Venerupis_philippinarum:97.0,(Hadrurus:85.0,(Artemia:111.0,(Gea:86.5,(Yoldia_limatula:80.0,(Helobdella_robusta:90.0,(Crassostrea_virginica:114.0,(Hypochilus:64.5,(Rhiphicephalus:99.5,(Limulus:69.0,(Lottia_gigantea:80.5,(Siphonaria_pectinata:109.0,(Perotrochus_lucaya:76.5,(Scaphopods:86.0,(Thelichoris:71.5,(Aportus:95.0,(Centruoides:73.0,(Lingula_anatina:106.5,(Lymnaea_stagnalis:84.5,(Thermobius:89.5,(Littorina_littorea:76.5,(Mastigoproctus:70.5,(Ilyanassa_obsoleta:76.0,(Amblypygid:70.5,Pauroctonus:77.5)HTU34:3.5)HTU33:3.5)HTU35:4.5)HTU36:5.0)HTU37:4.0)HTU38:2.5)HTU39:5.5)HTU40:4.5)HTU41:8.0)HTU42:4.0)HTU43:2.5)HTU44:3.5)HTU45:3.0)HTU46:5.0)HTU47:3.0)HTU48:6.0)HTU49:6.5)HTU50:3.5)HTU51:9.5)HTU52:8.0)HTU53:8.5)HTU54:5.0)HTU55:5.0)HTU56:2.5)HTU57:7.5)HTU58:17.0)HTU59:11.5,(Mytilus_californianus:78.5,(Americhernus:115.5,Vonones:68.5)HTU61:3.0)HTU62:5.0)HTU60:14.0)HTU32[792.0];
(Alentus:67.0,((Venerupis_decussatus:114.5,(Chanbria:74.5,(Aplysia_californica:113.0,(Venerupis_philippinarum:96.5,(Hadrurus:85.0,(Artemia:111.0,(Gea:86.0,(Yoldia_limatula:80.0,(Helobdella_robusta:90.0,(Crassostrea_virginica:114.0,(Hypochilus:64.0,(Rhiphicephalus:100.5,(Limulus:69.5,(Lottia_gigantea:80.5,(Siphonaria_pectinata:109.0,(Perotrochus_lucaya:76.5,(Scaphopods:86.0,(Thelichoris:69.5,(Centruoides:72.5,(Lingula_anatina:106.5,(Lymnaea_stagnalis:84.5,(Thermobius:92.0,(Littorina_littorea:76.5,(Mastigoproctus:68.5,(Ilyanassa_obsoleta:75.5,(Amblypygid:72.0,Pauroctonus:76.0)HTU34:3.5)HTU33:4.5)HTU35:4.0)HTU36:4.5)HTU37:4.0)HTU38:2.5)HTU39:3.5)HTU40:6.0)HTU41:5.0)HTU42:2.5)HTU43:3.5)HTU44:3.0)HTU45:5.0)HTU46:3.0)HTU47:5.5)HTU48:6.5)HTU49:3.5)HTU50:9.5)HTU51:8.0)HTU52:9.0)HTU53:5.0)HTU54:5.0)HTU55:2.5)HTU56:7.5)HTU57:19.0)HTU58:16.5,(Mytilus_californianus:78.5,(Americhernus:113.5,(Aportus:81.5,Vonones:74.5)HTU60:5.0)HTU61:3.0)HTU62:3.0)HTU59:10.5)HTU32[777.0];
